source/common/Utils.cpp

```diff
@@ -36,7 +37,7 @@
 
 int g_TimeSecondsOnInit = 0;
 
-#if (!defined(USE_SDL) || defined(_WIN32)) && !defined(ANDROID)
+#if (!defined(USE_SDL) || defined(_WIN32)) && !defined(ANDROID) && !MC_TARGET_OS_MAC
```

Comment:
This define is weird, maybe should be ifdef _WIN32?

Reply:
How am I supposed to fix someone else's shitty undocumented logical macro?